doc/samples.md

```diff
@@ -102,5 +103,60 @@ To use the Azure DevOps Extension in a hosted agent using a Release Pipeline, ex
 6. Add the script - either via file or inline. For the example, the script has been included inline. 
 ![PowerShell](/doc/images/Script.PNG)
 
-The first line of the script installs the Azure CLI and adds the Azure DevOps Extension. 
+Including the inline script for reference
+```bash
+pip install --pre azure-cli==2.0.55 --extra-index-url https://azurecliprod.blob.core.windows.net/edgeaz --h
+az --version

```

Comment:
we will need `az --version` only for debugging and while debugging we might need --debug in other commands as well

Reply:
Considering it's a first glance into set up using YAML, are they required? The aim is to provide simplistic view to set things up.

Reply:
nope, not required